core/controllers/topic_editor_test.py

```diff
@@ -501,13 +497,11 @@ def test_editable_topic_handler_put_raises_error_with_invalid_name(self):
         self.login(self.CURRICULUM_ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
 
-        json_response = self.put_json(
+        self.put_json(
             '%s/%s' % (
                 feconf.TOPIC_EDITOR_DATA_URL_PREFIX, self.topic_id),
             change_cmd, csrf_token=csrf_token, expected_status_int=400)
 
-        self.assertEqual(json_response['error'], 'Name should be a string.')
-
```

Comment:
Why is this removed?

Reply:
Thanks for flagging, that is not intentional. Added it back.